infrastructure/ci/dev-platform/development/create-sse-frontend-pipeline.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
This can be done with a more general script.

Reply:
removed pipeline script to use dev platform provided script